lib/gds_api/saved_page_api.rb

```diff
@@ -0,0 +1,47 @@
+# Adapter for the Saved Page API
+# Authenticated with Account API
+#
+# @see https://github.com/alphagov/account-api
+# @api documented
+
+class GdsApi::SavedPageApi < GdsApi::AccountApi
+  # Look up all pages saved by a user in their Account
+  #
+  # @param [String] govuk_account_session Value of the session header
+  #
+  # @return [Hash] containing :saved_pages, an array of single saved page hashes
+  def get_saved_pages(govuk_account_session:)
+    get_json("#{endpoint}/api/saved-pages", auth_headers(govuk_account_session))
```

Comment:
The endpoint is `/api/saved_pages`

Reply:
Yeah hang on, no point reviewing this yet until the account api bit is set.
There have been changes over there i need to reflect here too.